src/App.tsx

```diff
@@ -1,27 +1,74 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 // or
 // import * as goodsAPI from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goodsAll, setGoodsAll] = useState<Good[]>([]);
+  const [goods5First, setGoods5First] = useState<Good[]>([]);
+  const [goodsRed, setGoodsRed] = useState<Good[]>([]);
```

Comment:
it would be better to have one common state to set the current array which depends on what button you click
```suggestion
  const [good, setGood] = useState<Good[]>([]);
```